### ISSUE TITLE
Docs example brackets don't match

### ISSUE THREAD

**Issue by edmcdonagh** [carltongibson/django-filter#505]
[Ordering Filter docs](https://django-filter.readthedocs.io/en/latest/ref/filters.html#ordering-filter)

```
    o = OrderingFilter(
        # tuple-mapping retains order
        fields=(
            ('username', 'account'),
            ('first_name', 'first_name'),
            ('last_name', 'last_name'),
        },

        # labels do not need to retain order
        field_labels={
            'username': 'User account',
        }
    )

```

fields opens with a standard bracket and closes with a curly bracket.

**Comment by carltongibson** [comment]
Fixed in 55c31cbb7cf55077369d0c3bc9e8bac424eaf996

**Comment by edmcdonagh** [comment]
Thanks
